docs/source/search/searching_the_stac_catalog.ipynb

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
<p>Recommend adding the function name <code>get_all_collections</code> to the second sentence.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>The sentence may not matter, I was just thinking that <code>get_all_collections</code> might be good to mention before the code block that uses it. </p>